docs/examples/sidenavigation/activeItemExample.js

```diff
@@ -0,0 +1,36 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { SideNavigation } from 'gestalt';
+
+export default function Example(): Node {
+  const reactRouterPath = '/sidenavigation';
+
+  return (
+    <SideNavigation accessibilityLabel="Active item example">
+      <SideNavigation.Section label="Navigation">
+        <SideNavigation.TopItem
+          href="https://gestalt.pinterest.systems/pageheader"
```

Comment:
I don't think we want to use real URLs for the examples, as it causes the user to navigate to the docs site within the Sandpack iframe, which is very confusing. Let's just use `href="#"` for all examples with links.

Reply:
yeah, makes sense

Reply:
DONE